lib/datadog/security/contrib/integration.rb

```diff
@@ -0,0 +1,35 @@
+module Datadog
+  module Security
+    module Contrib
+      # Base provides features that are shared across all integrations
+      module Integration
+        @registry = {}
```

Comment:
I see we have two registries now (one here and the other for `ddtrace` contrib integrations), which is fine.

Unless you have a strong reason against it, I think we should refactor the existing `ddtrace` registry code to allow for both to share any applicable logic (is there any?).

Reply:
Yeah, I went the fast route here to have an implementation that can run separately from the tracer, as well as better identify the common surface between products. The integration registry management should probably end up in core.